onboarding-manager/pkg/cloudinit/infra.cfg

```diff
@@ -272,4 +322,22 @@ datasource:
                 systemctl stop sshd
                 systemctl disable sshd
                 {{- end }}
+                sudo tee /etc/logrotate.d/ufw << 'END'
```

Comment:
Please use `write_files` module to insert the content below to `/etc/logrotate.d/ufw`